modules/ROOT/pages/policies-policy-types.adoc

```diff
@@ -4,7 +4,7 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: policy, custom, ootb, offline
 
-Policies can be default, custom, or automated. Each type of policy has its own scope, management, and usability.
+Policies can be default, custom, or automated. Each type of policy has its own scope, management, and usability. You can uniformly apply a policy to all APIs or you can apply policies to select API based on a criteria, which is known as resource-level policies.
```

Comment:
"or you can apply policies to select API based on a criteria, which is known as resource-level policies." -> "or you can apply a policy to a selected API based on a specific criteria. The latter feature is known as resource-level policies."

Reply:
This text is not there anymore in the file. Done! Thanks!